app/assets/stylesheets/components/home/curriculum_details.scss

```diff
@@ -21,7 +21,7 @@
       flex-direction: column;
     }
 
-    .curriculum-details-tile {
+    .course-badge-tile {
```

Comment:
@dm-murphy Turns out these were quite old, thankfully reasonably minor change otherwise it would have started to be falling out of scope a little. 

What do you think of this name? A bit more descriptive now

Reply:
@ChargrilledChook I think that's much more descriptive and easily distinguishable from `curriculum-details-title` 👍